testing/tcolors/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Joe Searle <joe@jsearle.net>
+# Maintainer: Joe Searle <joe@jsearle.net>
+pkgname=tcolors
+pkgver=0.3.1
+pkgrel=0
+pkgdesc="Commandline color picker and palette builder"
+url="https://github.com/bcicen/tcolors"
+arch="all"
+license="MIT"
+makedepends="go gzip"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/bcicen/tcolors/archive/v$pkgver.tar.gz"
+options="!check" # No tests
+
+build() {
+	gzip -f -k docs/man1/tcolors.1
```

Comment:
Why do you manually compress the manpage here? Abuild should do that do you in $pkgname-doc's default function.

Reply:
I wasn't aware it did this. I have removed the gzip and retested and its fine, thanks.